src/api/spec/jobs/measurements_job_spec.rb

```diff
@@ -0,0 +1,42 @@
+RSpec.describe MeasurementsJob do
+  before do
+    rabbit_mock = Class.new do
+      attr_reader :metrics
+
+      def initialize
+        @metrics = {}
+      end
+
+      def metrics_sent
+        @metrics
+      end
+
+      def send_to_bus(category, data)
+        @metrics[category] = [] unless @metrics[category]
+        @metrics[category] << data
+      end
+    end
+    stub_const('RabbitMock', rabbit_mock)
+  end
+
+  describe 'token workflows' do
+    let(:rabbit_mock) { RabbitMock.new }
+
+    it 'gathers some intelligence about our token workflow runs' do
+      allow(RabbitmqBus).to receive(:send_to_bus).with('metrics', 'group count=0')
+      allow(RabbitmqBus).to receive(:send_to_bus).with('metrics', 'user in_beta=0,in_rollout=0,censored=0,count=0,unconfirmed=0,confirmed=0,locked=0,deleted=0,subaccount=0')
+      allow(RabbitmqBus).to receive(:send_to_bus).with('metrics', 'token_workflow total_count=0,enabled=0,disabled=0,custom_configuration_path=0,custom_configuration_url=0,user_with_tokens=0,groups_sharing_tokens=0')
+      allow(RabbitmqBus).to receive(:send_to_bus).with('metrics', 'notification_count,channel=web value=0')
+      allow(RabbitmqBus).to receive(:send_to_bus).with('metrics', 'notification_count,channel=rss value=0')
+      allow(RabbitmqBus).to receive(:send_to_bus).with('metrics', 'event_subscription_count,default=true value=0')
+      allow(RabbitmqBus).to receive(:send_to_bus).with('metrics', 'beta_feature_count,feature=request_show_redesign,status=disabled value=0')
+      allow(RabbitmqBus).to receive(:send_to_bus).with('metrics', 'beta_feature_count,feature=content_moderation,status=disabled value=0')
+      allow(RabbitmqBus).to receive(:send_to_bus).with('metrics', 'beta_feature_count,feature=color_themes,status=disabled value=0')
+      allow(RabbitmqBus).to receive(:send_to_bus).with('metrics', 'beta_feature_count,feature=foster_collaboration,status=disabled value=0')
+      allow(RabbitmqBus).to receive(:send_to_bus).with('metrics', 'beta_feature_count,feature=labels,status=disabled value=0')
+      allow(RabbitmqBus).to receive(:send_to_bus).with('metrics', 'beta_feature_count,feature=request_index,status=disabled value=0')
+
+      MeasurementsJob.new.perform({ 'amqp_options' => true })
+    end
```

Comment:
```suggestion
    it 'gathers some intelligence about our token workflow runs' do
      allow(RabbitmqBus).to receive(:send_to_bus)

      MeasurementsJob.new.perform({ 'amqp_options' => true })

      expect(RabbitmqBus).to have_received(:send_to_bus)
        .with('metrics', 'token_workflow total_count=0,enabled=0,disabled=0,custom_configuration_path=0,custom_configuration_url=0,user_with_tokens=0,groups_sharing_tokens=0')
        .at_least(1).times
    end
```

What about this option?